src/elements/e-video.vue

```diff
@@ -97,108 +85,126 @@
 
       /**
        * Allows to set the video player width.
+       * width can either be responsive or hard coded.
+       * youtube standard -> width="560".
        */
       width: {
-        type: [String, Number],
-        default: 560,
+        type: String,
+        default: null,
       },
 
       /**
        * Allows to set the video player height.
+       * height can either be responsive or hard coded.
+       * youtube standard -> height="315".
        */
       height: {
-        type: [String, Number],
-        default: 315,
-      },
-
-      /**
-       * Defines the players size to be responsive 100% width to the parent container
-       */
-      responsive: {
-        type: Boolean,
-        default: false,
+        type: String,
+        default: null,
```

Comment:
suggestion: Better use `default: undefined`. Optional props seem to always allow the `undefined` type. Using `null` means you need to support `string | undefined | null` in your code.  If we use `undefined` only `string | undefined` is allowed.